lib/src/snapd_client.dart

```diff
@@ -1475,6 +1475,28 @@ class SnapdClient {
     return snaps;
   }
 
+  /// Convenience function to find a snap by its ID. Under the hood, this
+  /// function combines the [assertions] and [find] endpoints.
+  ///
+  /// If no snap is found, this function will return null.
+  Future<Snap?> findById(String snapId,
+      {String? series, String? remote}) async {
```

Comment:
Change to `{String series = '16', ...` - then the default value is shown to the API user. But does it make sense to have this series as default? Will this get hopelessly out of date?